index.html

```diff
@@ -41,4 +41,29 @@ <h2>I’am a Software Developer</h2>
                 <li><a href="#" class="icon5"><i class="fab fa-medium-m fa-2x"></i></a></li>
             </ul>
             </div> 
-    </section>
\ No newline at end of file
+    </section>
+    <section>
+      <div>
+          <h1>My Recent Work</h1>
+      </div>
+      <hr>
+      <div class="img">
+          <h2>Multi-Post Stories</h2>
+      </div>
+      <div class="text-wrap">
+         <p> A daily selection of privately personalized reads; 
+          no accounts or sign-ups required. has been the industry's 
+          standard dummy text ever since the 1500s, when an unknown printer 
+          took a standard dummy text.
+         </p>
+      </div>
+      <section class="button">
+              <button type="submit" value="Submit" class="btn-icon1">css</button>
+              <button type="submit" value="Submit" class="btn-icon2">html</button>
+              <button type="submit" value="Submit" class="btn-icon3">bootstrap</button>
+              <button type="submit" value="Submit" class="btn-icon4">Ruby</button>
+          </section>
+          <section class="view">
+          <button type="submit" value="Submit" class="see-project">See Project</button>
+      </section>
```

Comment:
- [ ] Kindly complete the rest of the project as seen in the template and project requirement✔
- Add the remaining work cards
![screenshot (45)](https://user-images.githubusercontent.com/44624138/144502362-798078a4-157e-4db1-ab8f-57ba0adf66ab.png)
- Add the about page
 ![screenshot (46)](https://user-images.githubusercontent.com/44624138/144502065-28d2d73a-9d64-45df-b5d8-d728f2c59581.png)